src/Kernel/Behavior.class.st

```diff
@@ -138,8 +138,8 @@ Behavior >> allInstVarNames [
 
 	| vars |
 	self superclass == nil
-		ifTrue: [vars := self instVarNames copy]	"Guarantee a copy is answered."
-		ifFalse: [vars := self superclass allInstVarNames , self instVarNames].
+		ifTrue: [vars := self slots collect: #name]	"Guarantee a copy is answered."
```

Comment:
I don't see the method #slots in Behavior?

Did I missed it?

I seed #slots and #slotNames in ClassDescription